velox/exec/HashProbe.cpp

```diff
@@ -998,10 +998,21 @@ void HashProbe::prepareFilterRowsForNullAwareJoin(
       filterInputColumnDecodedVector_.decode(
           *filterInput_->childAt(projection.outputChannel), filterInputRows_);
       if (filterInputColumnDecodedVector_.mayHaveNulls()) {
+        std::vector<uint64_t> nullsInActiveRows(bits::nwords(numRows));
```

Comment:
since we are already allocating a vector from malloc, can we instead use a selectivity vector here and abstract away the bit manipulation by using the Selectivity vector's utility functions instead?